__tests__/actions/socialLogin.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable prefer-promise-reject-errors */
```

Comment:
Great work @EmyRukundo, In order to follow naming conventions, each test file should be like this: `socialLogin.test.js`

Reply:
Thanks @eliemugenzi, I have corrected...